<?xml version="1.0"?>
<launch>
	<arg name="type"/>	
	<group if="$(eval 'rv_6s' in arg('type'))">	
		<param name="/Mitsubishi_robot/len_of_link1" value="0.35" />
		<param name="/Mitsubishi_robot/len_of_link2" value="0.085" />
		<param name="/Mitsubishi_robot/len_of_link3" value="0.28" />
		<param name="/Mitsubishi_robot/len_of_link4" value="0.1" />
		<param name="/Mitsubishi_robot/len_of_link5" value="0.315" />
		<param name="/Mitsubishi_robot/len_of_link6" value="0.085" />
		<param name="/Mitsubishi_robot/max_speed" value="1" />
	</group>
	<group if="$(eval 'rv_6sdl' in arg('type'))">	
		<param name="/Mitsubishi_robot/len_of_link1" value="0.35" />
		<param name="/Mitsubishi_robot/len_of_link2" value="0.085" />
		<param name="/Mitsubishi_robot/len_of_link3" value="0.38" />
		<param name="/Mitsubishi_robot/len_of_link4" value="0.1" />
		<param name="/Mitsubishi_robot/len_of_link5" value="0.425" />
		<param name="/Mitsubishi_robot/len_of_link6" value="0.085" />
		<param name="/Mitsubishi_robot/max_speed" value="0.3" />
	</group>
	<group if="$(eval 'pg70' in arg('type'))">	
		<param name="/Mitsubishi_robot/len_of_gripper" value="0.132" />
	</group>
	<group unless="$(eval 'pg70' in arg('type'))">
		<param name="/Mitsubishi_robot/len_of_gripper" value="0." />
	</group>
	<node name="mitsubishi_kinematics_server" pkg="mitsubishi_arm_kinematics" type="mitsubishi_kinematics_server.py" respawn="false" output="screen" required="True"/>

</launch>
